torch/testing/_internal/common_distributed.py

```diff
@@ -318,6 +321,10 @@ def tearDown(self):
         # processes to prevent an effective file descriptor leak.
         self.processes = []
 
+        # Close all the files
```

Comment:
never used faulthandler before, is `faulthandler.disable()` also needed?

Reply:
I don't think it is necessary, you need to do it only if you want to disable the signal handlers at some point.